lib/public/AppFramework/Http/DataDisplayResponse.php

```diff
@@ -41,17 +45,14 @@ class DataDisplayResponse extends Response {
 
 	/**
 	 * @param string $data the data to display
-	 * @param int $statusCode the Http status code, defaults to 200
-	 * @param array $headers additional key value based headers
+	 * @param S $statusCode the Http status code, defaults to 200
+	 * @param H $headers additional key value based headers
 	 * @since 8.1.0
 	 */
-	public function __construct($data = '', $statusCode = Http::STATUS_OK,
-								$headers = []) {
-		parent::__construct();
+	public function __construct($data = '', int $statusCode = Http::STATUS_OK, array $headers = []) {
```

Comment:
Either touching is okay, then:
```suggestion
	public function __construct(string $data = '', int $statusCode = Http::STATUS_OK, array $headers = []) {
```

otherwise since the parameters existed here already:
```suggestion
	public function __construct($data = '', $statusCode = Http::STATUS_OK, $headers = []) {
```

Reply:
Fixed